### PR TITLE
Add - My Glucose Manager - follow your daily health

### PR DESCRIPTION
**Pull request by foxmask**

### CODE REVIEW

**Comment by Kickball** [comment]
Please update this to include the license and programming language as per our standard format.

``[Name](http://homepage/) - Short description ([Demo](http://url.to/demo), [Source code](http://url.of/source/code)) `License` `Language` ``

**Comment by foxmask** [comment]
sorry for forgetting

**Comment by Kickball** [comment]
@nodiscc can you give this a final checkover when you get time please.

File: README.md [hunk 1]
```diff
@@ -524,11 +524,12 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
 
 ## Monitoring and Administration
 
+  * [Dj Diabetes](https://github.com/foxmask/dj-diabetes) - My Glucose Manager - follow your daily health  `BSD` `Python`
   * [Prometheus](http://prometheus.io/) - Monitoring system and time series database. ([Source code](https://github.com/prometheus/prometheus)) `Apache` `Go`
   * [psdash](https://github.com/Jahaja/psdash) - A linux system information web dashboard using psutils and flask
   * [pyDash](https://github.com/k3oni/pydash) - Small web-based monitoring dashboard for linux ([Demo](http://pydash.hostechs.com/main/) - _admin/admin_)
   * [sensu](https://sensuapp.org/) - monitor servers, services, application health, and business KPIs. Get notified about failures before your users do. Collect and analyze custom metrics. `MIT`
-   * [uchiwa](https://uchiwa.io/) - simple dashboard for sensu
+  * [uchiwa](https://uchiwa.io/) - simple dashboard for sensu
```
Comment (nodiscc) [inline README.md]: @foxmask please do not remove the indentation here (uchiwa should be indented under sensu)

**Comment by nodiscc** [comment]
Along with the previous comment, I think the proper category is https://github.com/Kickball/awesome-selfhosted/#personal-dashboards

**Comment by foxmask** [comment]
I hope this will do the trick :)

File: README.md [hunk 2]
```diff
@@ -527,8 +527,8 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
   * [Prometheus](http://prometheus.io/) - Monitoring system and time series database. ([Source code](https://github.com/prometheus/prometheus)) `Apache` `Go`
   * [psdash](https://github.com/Jahaja/psdash) - A linux system information web dashboard using psutils and flask
   * [pyDash](https://github.com/k3oni/pydash) - Small web-based monitoring dashboard for linux ([Demo](http://pydash.hostechs.com/main/) - _admin/admin_)
-  * [sensu](https://sensuapp.org/) - monitor servers, services, application health, and business KPIs. Get notified about failures before your users do. Collect and analyze custom metrics. `MIT`
-   * [uchiwa](https://uchiwa.io/) - simple dashboard for sensu
+   * [sensu](https://sensuapp.org/) - monitor servers, services, application health, and business KPIs. Get notified about failures before your users do. Collect and analyze custom metrics. `MIT`
+  * [uchiwa](https://uchiwa.io/) - simple dashboard for sensu
```
Comment (nodiscc) [inline README.md]: @foxmask indentation is still wrong :)

 * sensu
  * uchiwa

**Comment by foxmask** [comment]
ok ; I redo everything from the beginning - shame one me ...

**Pull request closed without merging**